src/py42/clients/savedsearch.py

```diff
@@ -32,17 +32,19 @@ def get_by_id(self, search_id):
         uri = u"{}/{}".format(self._resource, search_id)
         return self._session.get(uri)
 
-    def get_query(self, search_id):
+    def get_query(self, search_id, pg_num=1, pg_size=10000):
```

Comment:
Since this was working before without the need for passing values here, we should default to `None` and let the other py42 mechanisms figure out the `pg_num` and `pg_size` values.  This way, if we ever need to change 10,000 to something else, we don't have to fix the bug in multiple places.

Reply:
Ah, yes. I did consider that, but didn't change it. I'll update. Thanks!